src/renderer/components/SettingsModal.tsx

```diff
@@ -206,6 +208,25 @@ const AppearanceSettings = memo(() => {
                     </NumberInputStepper>
                 </NumberInput>
             </SettingsItem>
+
+            <SettingsItem
+                description="The amount of padding for the viewport PNG export."
+                title="Viewport PNG export padding"
+            >
+                <NumberInput
+                    defaultValue={viewportExportPadding || 20}
```

Comment:
According to the type, `viewportExportPadding` is a number, so what does this `|| 20` mean?

Also, why is the default value dependent on the current value? Did you mean for the default to simply be 20?

Reply:
Well i actually just copied the setup from the snapToGridAmount settings above. Which uses the following:
`defaultValue={snapToGridAmount || 1}`

But I just tested it without all it, and it works as expected.
Will change that.

Reply:
We may want to look into why we even have that for snapToGridAmount... That's probably a holdover from before we used typescript and I was just trying to make sure nothing errored.